expenseease-group-svc/src/main/java/com/expenseease/group/controller/GroupController.java

```diff
@@ -39,4 +39,16 @@ public Set<UserDTO> findUsersInGroup(@RequestHeader("Authorization") String toke
                                          @PathVariable Long groupId) {
         return groupService.findUsersInGroup(groupId);
     }
+
+    @DeleteMapping("/groups/{groupId}")
+    public void deleteGroup(@RequestHeader("Authorization") String token,
+                            @PathVariable Long groupId){
+        groupService.deleteGroup(groupId);
+    }
+
+    @PutMapping("/groups/{userId}/{groupId}")
+    public GroupDTO exitFromGroup(@RequestHeader("Authorization") String token,
+                                  @PathVariable Long userId, @PathVariable Long groupId){
+        return groupService.removeUserFromGroup(userId, groupId);
```

Comment:
To exit from the group, we will not receive the userId in the request.
We need to find the userId from the currently loggedIn user.
Refer to createGroup.
We can extract the userId like below:
`Authentication authentication = SecurityContextHolder.getContext().getAuthentication();User loggedInUser = (User) authentication.getPrincipal();`